pkg/minishift/openshift/oc.go

```diff
@@ -42,3 +47,38 @@ func AddSudoersRoleForUser(user string) error {
 	}
 	return nil
 }
+
+// Get the route for service
+func GetServiceURL(service, namespace string) (string, error) {
+	cmdName := instanceState.Config.OcPath
+	cmdArgText := fmt.Sprintf("get route/%s", service)
+	if namespace != "" && !isProjectExists(namespace) {
```

Comment:
No need to check also the namespace here...
likely bail out when namespace == ""

or is an empty namespace something we can query?